### PR TITLE
FrostWireLLC.FrostWire version 6.9.9.313

### PR DESCRIPTION
**Pull request by ptanmay143**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/83320)

### CODE REVIEW

**Comment by ptanmay143** [comment]
[Package Request]: FrostWireLLC.FrostWire #83137

File: manifests/f/FrostWireLLC/FrostWire/6.9.9.313/FrostWireLLC.FrostWire.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: FrostWireLLC.FrostWire
+PackageVersion: 6.9.9.313
```
Comment (KaranKad) [inline manifests/f/FrostWireLLC/FrostWire/6.9.9.313/FrostWireLLC.FrostWire.installer.yaml:5]:
```suggestion
PackageIdentifier: FrostWire.FrostWire
PackageVersion: 6.9.9.313
InstallModes:
- interactive
- silent
ReleaseDate: 2022-09-12
Scope: machine
```

File: manifests/f/FrostWireLLC/FrostWire/6.9.9.313/FrostWireLLC.FrostWire.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: FrostWireLLC.FrostWire
```
Comment (KaranKad) [inline manifests/f/FrostWireLLC/FrostWire/6.9.9.313/FrostWireLLC.FrostWire.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: FrostWire.FrostWire
```

File: manifests/f/FrostWireLLC/FrostWire/6.9.9.313/FrostWireLLC.FrostWire.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: FrostWireLLC.FrostWire
+PackageVersion: 6.9.9.313
+PackageLocale: en-US
+Publisher: FrostWire LLC
+PackageName: FrostWire
+License: GPL
```
Comment (KaranKad) [inline manifests/f/FrostWireLLC/FrostWire/6.9.9.313/FrostWireLLC.FrostWire.locale.en-US.yaml:9]:
```suggestion
License: GPL-3.0
LicenseUrl: https://raw.githubusercontent.com/frostwire/frostwire/master/desktop/LICENSE.txt
```

File: manifests/f/FrostWireLLC/FrostWire/6.9.9.313/FrostWireLLC.FrostWire.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: FrostWireLLC.FrostWire
+PackageVersion: 6.9.9.313
+PackageLocale: en-US
+Publisher: FrostWire LLC
+PackageName: FrostWire
+License: GPL
+Copyright: FrostWire LLC 2019
+ShortDescription: FrostWire - Search, Download, Play, Share.
```
Comment (KaranKad) [inline manifests/f/FrostWireLLC/FrostWire/6.9.9.313/FrostWireLLC.FrostWire.locale.en-US.yaml:11]:
```suggestion
ShortDescription: FrostWire is an easy to use Cloud Downloader, BitTorrent Client and Media Player.
```

File: manifests/f/FrostWireLLC/FrostWire/6.9.9.313/FrostWireLLC.FrostWire.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: FrostWireLLC.FrostWire
```
Comment (KaranKad) [inline manifests/f/FrostWireLLC/FrostWire/6.9.9.313/FrostWireLLC.FrostWire.yaml:4]:
```suggestion
PackageIdentifier: FrostWire.FrostWire
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

**Review by KaranKad — APPROVED** [review]

**Pull request merged**